docker-compose.yml

```diff
@@ -147,7 +147,7 @@ services:
     command: ['yarn', 'joystream-distributor', 'start']
 
   db:
-    image: postgres:12
+    image: postgres:14
```

Comment:
Thank you very much for spotting this issue, I guess I didn't fully evaluate all the consequences of switching to `v14`.
The reason I did that was because I noticed that some SQL function used by one of the Orion queries was missing in `v12` and I realized that we use `v14` in the Orion repo. Switching to `v14` in the monorepo as well fixed this particular issue.

I think the easiest solution would be to run 2 separate db services like `db` and `orion-db`, where `db` is `v12` and used only by the query node, while `orion-db` would be `v14` and would be only used by Orion.